cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -280,9 +280,11 @@ function Initialize-Environment() {
   # show notification to change execution policy:
   $allowedExecutionPolicy = @('Unrestricted', 'RemoteSigned', 'ByPass')
   If ((Get-ExecutionPolicy).ToString() -notin $allowedExecutionPolicy) {
-    Write-Error "PowerShell requires an execution policy in [$($allowedExecutionPolicy -join ", ")] to run $app_name."
-    Write-Error "For example, to set the execution policy to 'RemoteSigned' please run :"
-    Write-Error "'Set-ExecutionPolicy RemoteSigned -scope CurrentUser'"
+    throw @"
+PowerShell requires an execution policy in [$($allowedExecutionPolicy -join ", ")] to run $app_name.
+For example, to set the execution policy to 'RemoteSigned' please run :
+'Set-ExecutionPolicy RemoteSigned -scope CurrentUser'
+"@
     break
```

Comment:
```suggestion
    throw @"
Error: PowerShell requires an execution policy in [$($allowedExecutionPolicy -join ", ")] to run $app_name. For example, to set the execution policy to 'RemoteSigned' please run:

    Set-ExecutionPolicy RemoteSigned -scope CurrentUser

"@
```

Is more readable, but the interactive error handler eats the whitespace, so see other comments